src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -1,28 +1,23 @@
 package io.github.thebusybiscuit.slimefun4.utils;
 
+import java.net.MalformedURLException;
+import java.net.URL;
 import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.Base64;
-import java.util.List;
-import java.util.Optional;
-import java.util.OptionalInt;
+import java.util.*;
```

Comment:
dont star import

Reply:
apologies, my IDE did that. I've fixed it.